lib/bulbs-elements/util/prep-ga-event.js

```diff
@@ -0,0 +1,18 @@
+export default function prepGaEvent (gaPrefix, gaId, dimensions = {}) {
```

Comment:
since the function is returning a function that calls a ga event, I wanted to make that clear in the function name, maybe `prepGaEventTracker`?